gnomad_constraint/utils/generic.py

```diff
@@ -244,3 +244,34 @@ def remove_unnecessary_variants(
         exome_crit &= exome_ht.freq[freq_index].AF <= af_cutoff
     exome_ht = exome_ht.filter(exome_crit)
     return context_ht, exome_ht
+
+
+def get_all_pop_lengths(
+    ht, pops: Tuple[str], prefix: str = "observed_", skip_assertion: bool = False
+):
+    """
+    Get the minimum array length for specific per population annotations in `ht`.
+
+    The annotations are specified by the combination of `prefix` and each population in `pops`.
+
+    :param ht: Input Table.
+    :param pops: List of populations. Defaults to `POPS`.
```

Comment:
this says List but the function args type this as a Tuple

Reply:
Tuple is just to make list unmutatble, so I guess it won't cause any confusion. It's consistent with functions in previous part like `create_constraint_training_dataset`